src/client-side-encryption/providers/index.ts

```diff
@@ -122,34 +122,31 @@ export interface KMSProviders {
    * Configuration options for using 'aws' as your KMS provider
    */
   aws?: AWSKMSProviderConfiguration | Record<string, never>;
+  [key: `aws:${string}`]: AWSKMSProviderConfiguration;
```

Comment:
Note: named kms providers do not support automatic KMS credential fetching, so no `Record<string, never>`.